crates/turbopack-ecmascript-runtime/js/src/build/runtime.ts

```diff
@@ -40,11 +42,32 @@ type ModuleFactory = (
 
 const path = require("path");
 const relativePathToRuntimeRoot = path.relative(RUNTIME_PUBLIC_PATH, ".");
+// Compute the relative path to the `distDir`.
+const relativePathToDistRoot = path.relative(path.join(OUTPUT_ROOT, RUNTIME_PUBLIC_PATH), ".");
```

Comment:
`OUTPUT_ROOT` here is likely `PROJECT/.next`, no? In order to use the input paths, don't we need `PROJECT`? The calculation here depends on the output path being permanently locked to `.next`, which isn't the case because of `NextConfig.distDir` configuration.

Reply:
No, we pass it down from https://github.com/vercel/turbo/pull/6318/files/b5ee04c02ee1eeabd70091397de29f8b1f446220#diff-22085854b7935369bed1065b0669564cf7d2a0287b06e0bc31164f2138a9277eR51, means whatever we specify as distDist should be embedded correctly. If it's fixed to `./next` somehow, we have a bugs to setting output_root in the chunk context configuration.

Reply:
both of the global `OUTPUT_ROOT` and `RUNTIME_PUBLIC_PATH` is being embedded from chunk context's configuration, so calculation should honor custom values if defined.